### ISSUE TITLE
`login': [!] undefined method `login' for Spaceship::Tunes:Module (NoMethodError)

### ISSUE THREAD

**Issue by irfanG** [fastlane/fastlane#11496]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
I get this error when trying to make build to testflight. 

/Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/pilot/lib/pilot/manager.rb:22:in `login': [!] undefined method `login' for Spaceship::Tunes:Module (NoMethodError)
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/pilot/lib/pilot/manager.rb:15:in `start'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/pilot/lib/pilot/build_manager.rb:12:in `upload'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/actions/upload_to_testflight.rb:16:in `run'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/runner.rb:253:in `block (2 levels) in execute_action'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/runner.rb:231:in `block in execute_action'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/runner.rb:227:in `chdir'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/runner.rb:227:in `execute_action'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/runner.rb:148:in `trigger_action_by_name'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/fast_file.rb:148:in `method_missing'
	from Fastfile:18:in `block (2 levels) in parsing_binding'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/lane.rb:33:in `call'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/runner.rb:45:in `chdir'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/runner.rb:45:in `execute'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/lane_manager.rb:56:in `cruise_lane'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/command_line_handler.rb:34:in `handle'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/commands_generator.rb:107:in `block (2 levels) in run'
	from /Library/Ruby/Gems/2.3.0/gems/commander-fastlane-4.4.5/lib/commander/command.rb:178:in `call'
	from /Library/Ruby/Gems/2.3.0/gems/commander-fastlane-4.4.5/lib/commander/command.rb:153:in `run'
	from /Library/Ruby/Gems/2.3.0/gems/commander-fastlane-4.4.5/lib/commander/runner.rb:476:in `run_active_command'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:75:in `run!'
	from /Library/Ruby/Gems/2.3.0/gems/commander-fastlane-4.4.5/lib/commander/delegates.rb:15:in `run!'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/commands_generator.rb:336:in `run'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/commands_generator.rb:42:in `start'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/cli_tools_distributor.rb:94:in `take_off'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/bin/fastlane:20:in `<top (required)>'
	from /usr/local/bin/fastlane:23:in `load'
	from /usr/local/bin/fastlane:23:in `<main>'
##### Complete output when running fastlane, including the stack trace and command used
/Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/pilot/lib/pilot/manager.rb:22:in `login': [!] undefined method `login' for Spaceship::Tunes:Module (NoMethodError)
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/pilot/lib/pilot/manager.rb:15:in `start'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/pilot/lib/pilot/build_manager.rb:12:in `upload'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/actions/upload_to_testflight.rb:16:in `run'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/runner.rb:253:in `block (2 levels) in execute_action'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/runner.rb:231:in `block in execute_action'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/runner.rb:227:in `chdir'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/runner.rb:227:in `execute_action'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/runner.rb:148:in `trigger_action_by_name'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/fast_file.rb:148:in `method_missing'
	from Fastfile:18:in `block (2 levels) in parsing_binding'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/lane.rb:33:in `call'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/runner.rb:45:in `chdir'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/runner.rb:45:in `execute'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/lane_manager.rb:56:in `cruise_lane'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/command_line_handler.rb:34:in `handle'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/commands_generator.rb:107:in `block (2 levels) in run'
	from /Library/Ruby/Gems/2.3.0/gems/commander-fastlane-4.4.5/lib/commander/command.rb:178:in `call'
	from /Library/Ruby/Gems/2.3.0/gems/commander-fastlane-4.4.5/lib/commander/command.rb:153:in `run'
	from /Library/Ruby/Gems/2.3.0/gems/commander-fastlane-4.4.5/lib/commander/runner.rb:476:in `run_active_command'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:75:in `run!'
	from /Library/Ruby/Gems/2.3.0/gems/commander-fastlane-4.4.5/lib/commander/delegates.rb:15:in `run!'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/commands_generator.rb:336:in `run'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/commands_generator.rb:42:in `start'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/fastlane/lib/fastlane/cli_tools_distributor.rb:94:in `take_off'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.75.0/bin/fastlane:20:in `<top (required)>'
	from /usr/local/bin/fastlane:23:in `load'
	from /usr/local/bin/fastlane:23:in `<main>'
[INSERT OUTPUT HERE]

### Environment
<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 10.13.2                                                        |
| Ruby                        | 2.3.3                                                          |
| Bundler?                    | false                                                          |
| Git                         | git version 2.14.2                                             |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | Mac OS X 10.13.2 (17C88)                                       |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib |
| OpenSSL Version             | LibreSSL 2.2.7                                                 |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                    |
| Xcode Version               | 9.2                                                            |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# This file contains the fastlane.tools configuration
# You can find the documentation at https://docs.fastlane.tools
#
# For a list of all available actions, check out
#
#     https://docs.fastlane.tools/actions
#

# Uncomment the line if you want fastlane to automatically update itself
#update_fastlane

default_platform(:ios)

platform :ios do
  desc "Push a new beta build to TestFlight"
  lane :beta do
	build_app(workspace: "Hermes.xcworkspace", scheme: "Hermes Dev")
    upload_to_testflight
  end
end

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier "is.symphony.HermesDev" # The bundle identifier of your app
apple_id "irfan.godinjak@symphony.is" # Your Apple email address

itc_team_id "450158" # iTunes Connect Team ID
team_id "3KL9338JP6" # Developer Portal Team ID

# For more information about the Appfile, see:
#     https://docs.fastlane.tools/advanced/#appfile

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.75.0  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.0.0        |
| slack-notifier            | 1.5.1        |
| CFPropertyList            | 2.3.6        |
| claide                    | 1.0.2        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.5.4        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.2.8        |
| terminal-notifier         | 1.8.0        |
| unicode-display_width     | 1.3.0        |
| terminal-table            | 1.8.0        |
| plist                     | 3.4.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.2        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.6.4        |
| tty-cursor                | 0.5.0        |
| tty-spinner               | 0.7.0        |
| babosa                    | 1.0.2        |
| colored                   | 1.2          |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.5        |
| excon                     | 0.60.0       |
| faraday                   | 0.13.1       |
| unf_ext                   | 0.0.7.4      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| fastimage                 | 2.1.1        |
| gh_inspector              | 1.0.3        |
| json                      | 1.8.3        |
| mini_magick               | 4.5.1        |
| multi_json                | 1.13.0       |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.2.1        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 1.0.0        |
| dotenv                    | 2.2.1        |
| bundler                   | 1.16.1       |
| faraday_middleware        | 0.12.2       |
| uber                      | 0.1.0        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.1        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| little-plugger            | 1.1.4        |
| logging                   | 2.2.2        |
| jwt                       | 2.1.0        |
| memoist                   | 0.16.0       |
| os                        | 0.9.6        |
| signet                    | 0.8.1        |
| googleauth                | 0.6.2        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.13.6       |
| io-console                | 0.4.5        |
| libxml-ruby               | 2.9.0        |
</details>

*generated on:* **2018-01-09**
</details>

[INSERT OUTPUT HERE]

**Comment by irfanG** [comment]
need to add 
get_certificates           # invokes cert
get_provisioning_profile

**Comment by andrey-lisovskiy** [comment]
doesn't help

**Comment by irfanG** [comment]
you need to create certificates etc as per described in the https://codesigning.guide guide. Once that is done properly, then your lane needs to look something like: 
```platform :ios do
  desc "Push a new beta build to TestFlight"
  lane :beta do
  	get_certificates           # invokes cert
  	get_provisioning_profile
	build_app(workspace: "App.xcworkspace", scheme: "App")
    upload_to_testflight
  end
end
```

**Comment by kurtheiligmann** [comment]
I am having the same issue when trying to download metadata; no certificate involved. 
Possible duplicate of #11495

**Comment by KrauseFx** [comment]
Sorry about that, shipping a new release right now 👍
